README.md

```
@@ -1,2 +1,59 @@
+```bash
+EMAIL_PRO=<pro-test-email>
+PASSWORD_PRO=<pro-test-password>
+ENVIRONMENT=<env>
+SLACK_BOT_USER_OAUTH_TOKEN=token
+```
+
+- <env> is the environment where the tests will run. It can be `prod`, `test`, or `dev`.
+- <SLACK_BOT_USER_OAUTH_TOKEN> is the Slack bot user OAuth token for the channel you want to send the test results to.
+- <pro-test-email> and <pro-test-password> are the credentials for a user with PRO rights.
+
+3. Run the api tests
```

Markdownlint

Expected: 1; Actual: 3; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)